src/DraggableList.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2010, Sage Software, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import declare from 'dojo/_base/declare';
+import List from './List';
```

Comment:
I believe the reason I did this was because the _ConfigureBase used List, should I include both List and DraggableList for the imports and make the DraggableList extend _ListBase?